updater/docs/01-Generate.md

```diff
@@ -0,0 +1,80 @@
+  # generated as well as the standard templates.
+  --uses-js \
+  --owner purescript-contrib \
+  --main-branch master \
+  --display-name '`argonaut-codecs`' \
+  --title 'Argonaut Codecs' \
+  --maintainer thomashoneyman
+```
+
+In typical usage you will:
+
+1. Clone the target repository and change into it
+2. Run `contrib-updater generate` and provide values for the relevant variables; if the library uses any JS files, include the `--uses-js` flag.
+3. Reconcile the new content with any information that should be preserved from the existing repository (any conflicting files will have been added to the backups directory). For example, you may want to copy sections of the old README into the new one or into the documentation where appropriate.
+4. Delete the backups directory and any other files which are no longer necessary in the target repository.
+5. Open a PR with your changes!
+
+## Variables
+
+Templates can use the following variables (in code, see the `Updater.Variable` module), provided via the `contrib-updater` tool:
+
+```purs
+type Variables =
+  { owner :: String
+  , mainBranch :: String
+  , packageName :: String
+  , displayName :: String
+  , displayTitle :: String
+  , maintainer :: String
+  }
+```
+
+- `owner` refers to the owner of the repository being updated. Defaults in the CLI to `"purescript-contrib"`.
+- `mainBranch` refers to the primary branch used in the repository. Defaults in the CLI to `"main"`, but many libraries will need to use `"master"` instead.
+- `packageName` refers to the package name as represented in the PureScript registry and Spago installation instructions. This is pulled automatically from the `spago.dhall` file.
+- `displayName` refers to the way you'd like to render the package name in markdown files. Defaults in the CLI to the name of the package in backticks, ie. `argonaut-codecs`, but it's also common to provide a string (for example, Argonaut Codecs) instead.
+- `displayTitle` refers to the way you'd like to render the package name in markdown titles. Defaults in the CLI to the name of the package without backtics, ie. argonaut-codecs, but it's also common to provide a string (for example, Argonaut Codecs) instead.
+- `maintainer` refers to the assigned maintainer for the library (ex: `"thomashoneyman"`). This is required in the CLI.
+
+Any of these variables can be used in template files via `{{variableName}}` syntax. When templates are generated for a particular repository these variables will be replaced with the values you provided.
+
+## Provided Templates
+
+The templates include default content that each Contributor library is expected to have. Templates are provided in layers, where later layers override earlier ones:
+
+### Layer 1: Base
+
+`base` includes content and configuration each Contributor library is expected to have.
+
+- An informative README with badges for the build status, latest release, latest Pursuit documentation, and current maintainers.
```

Comment:
Multiple maintainers?

Reply:
Mm -- Should be `maintainer(s)`, as it's not necessary to have more than one (though it is desirable).